service-app/internal/logger/logger.go

```diff
@@ -0,0 +1,41 @@
+package logger
+
+import (
+	"log"
+
+	"github.com/ministryofjustice/opg-scanning/config"
+	"github.com/ministryofjustice/opg-scanning/internal/util"
+)
+
+type Logger struct {
+	cfg         *config.Config
+	WriteToFile func(fileName string, message string, path string)
+}
+
+func NewLogger() *Logger {
```

Comment:
We have [a structured log standard](https://docs.opg.service.justice.gov.uk/documentation/adrs/adr-009.html) across OPG products.

The easiest way to do that is building a pre-configured logger from our shared go library: [example in another repo](https://github.com/ministryofjustice/opg-data-lpa-store/blob/49fd248935f659d071d556804b7fa90c17aa9f93/lambda/getlist/main.go#L86). You can then [use the methods on it](https://github.com/ministryofjustice/opg-data-lpa-store/blob/49fd248935f659d071d556804b7fa90c17aa9f93/lambda/update/main.go#L117) and it will take care of adding level/time/service fields plus any extras you supply as arguments.

Reply:
Well noted, I have implemented the OPG logger into the project.